packages/airnode-admin/src/cli.ts

```diff
@@ -448,7 +448,13 @@ yargs
     'Generates a random mnemonic. Uses "ethers.Wallet.createRandom" under the hood.',
     async () => {
       const mnemonic = await admin.generateMnemonic();
-      console.log(`Generated mnemonic: ${mnemonic}`);
+      const lines = [
+        'This mnemonic is created locally on your machine using "ethers.Wallet.createRandom" under the hood.',
+        'Make sure to back it up securely, e.g., by writing it down on a piece of paper:',
+        '',
+        mnemonic,
```

Comment:
We could maybe make some of the outputs a bit nicer using chalk? Not in the scope of this PR, but something to consider. What do you think?

```suggestion
        chalk.red(mnemonic),
```

Reply:
I wouldn't recommend that for the data that should be copied as it can copy the coloring markers from the terminal (e.g. `\033[0;31m`) and therefore broke the pasted output.